### ISSUE TITLE
[Bug] Packaged technology disappears

### ISSUE THREAD

**Issue by satotatsu5** [zencq/NomNom#122]

**Comment by satotatsu5** [comment]
It happens just by packaging it in-game and saving it after loading it with NomNom

**Comment by Pd69bq** [comment]
yeah, experienced the same issue with NomNom editor. and sometimes my capital freighter class will (re)set to lowest class C and remove all the perks on one of the frigate in my fleet

**Comment by peter3** [comment]
seeing the same with packaged technology with latest version

**Comment by deepspaceaxolotl** [comment]
Still experiencing this issue on the latest version, with packaged modules (though I think I had a Living Ship base technology archive disappear too, can't try reproducing now).

**Comment by deepspaceaxolotl** [comment]
Checking the backup of my save that I made, they were S- and X-class modules, while the packaged base tech modules (in this case, cargo scan protection and the economy scanner, both for the Living Ship) I think survived, though they don't display properly in the inventory while using the program.

**Comment by cengelha** [comment]
@flarejoltporeon That helped, thanks! I ended up adding all for testing and now all (known) should be covered. If you still find one that disappears (after game updates wait for a NN update first), let me know!
